src/lighteval/models/model_config.py

```diff
@@ -222,6 +222,7 @@ class InferenceEndpointModelConfig:
     should_reuse_existing: bool = False
     add_special_tokens: bool = True
     revision: str = "main"
+    namespace: str = None
```

Comment:
Would be nice to have a little documentation. I found myself mistaking this `namespace` arg for the `Namespace` class from the `argparse` package